gossip/src/crds_value.rs

```diff
@@ -160,13 +160,13 @@ impl CrdsData {
 }
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, AbiExample)]
-pub struct SnapshotHash {
+pub struct SnapshotHashes {
     pub from: Pubkey,
     pub hashes: Vec<(Slot, Hash)>,
     pub wallclock: u64,
 }
```

Comment:
And here's the actual struct. The singular "hash" was a little strange to me, since there's a vector of 'em.